src/components/Toaster/Toast/Toast.tsx

```diff
@@ -209,8 +210,10 @@ export function Toast(props: ToastUnitedProps) {
             {...heightProps}
             {...closeOnTimeoutProps}
         >
+            <div className={b('icon-container')}>
+                {renderIcon ? renderIcon(props) : renderIconByType({type})}
+            </div>
```

Comment:
Why icon wrapped to div and moved out of the container?

Reply:
In the old layout, the size of the icon was implicitly declared in the padding.
It means - we have to provide icons with the same size.
When I discussed the possibility of customizing icons with @korvin89 He advised me to also get away from the predefined size.
So I use flex container and` icon-container` should stretch to the size of the passed icon.